src/probnum/statespace/generate_samples.py

```diff
@@ -15,10 +18,6 @@ def generate_samples(dynmod, measmod, initrv, times):
         Random variable according to initial distribution
     times : np.ndarray, shape (n,)
         Timesteps on which the states are to be sampled.
```

Comment:
Do we want to add `random_state` to the parameter doc string?

Reply:
Done